src/main/java/com/volasoftware/tinder/service/UserServiceImpl.java

```diff
@@ -169,4 +144,16 @@ public UserProfileDto editUserProfile(@RequestBody UserProfileDto userProfileDto
 
         return new UserProfileDto(user.getFirstName(), user.getLastName(), user.getEmail(), user.getGender());
     }
+
+    public void getNewGeneratedPassword(String email) throws MessagingException, IOException {
+        User user = userRepository.findOneByEmail(email).orElseThrow(
+                () -> new UserDoesNotExistException("User with this email does not exist"));
+
+        String characters = "abcdefghijklmnopqrstuvwxyz0123456789";
```

Comment:
Can you find another way to say "use all small letters in English and all digits". Also move this to utility class or helper one, because the code is just to create some random password, the benefit doing that is reusability